src/literals.py

```diff
@@ -33,24 +33,14 @@
 # LOGS_RULES_DIR = "./src/alert_rules/loki"
 
 SUBSTRATE = "vm"
-USER = "root"  # TODO change to snap user
-GROUP = "root"
+USER = "snap_daemon"
+GROUP = "snap_daemon"
 
-# FIXME create expected paths once snap is integrated
 PATHS = {
-    "CONF": "/etc/karapace",
-    "DATA": "",
-    "BIN": "",
-    "LOGS": "",
+    "CONF": f"/var/snap/{SNAP_NAME}/current/etc/karapace",
+    "LOGS": f"/var/snap/{SNAP_NAME}/common/var/log/karapace",
```

Comment:
**suggestion** I believe these properties could just go directly to the `workload.py` file. I would only keep here the constant that are substrate independent